__tests__/pet.test.js

```diff
@@ -0,0 +1,177 @@
+    });
+
+    it('increments the age by 1', () => {
+        const pet = new Pet('Fido');
+
+        pet.growUp();
+
+        expect(pet.age).toEqual(1);
+    });
+
+    it('increments hunger by 5', () => {
+        const pet = new Pet('Fido');
+
+        pet.growUp();
+
+        expect(pet.hunger).toEqual(5);
+    });
+
+    it('decreases fitness by 3', () => {
+        const pet = new Pet('Fido');
+
+        pet.growUp();
+
+        expect(pet.fitness).toEqual(7);
+    });
+});
+
+describe('walk', () => {
+    it('throws an error if pet is not alive', () => {
+        const pet = new Pet('Fido')
+
+        pet.age = 30;
+
+        expect(() => pet.walk()).toThrow('Your pet is no longer alive')
+    });
+
+    it('increases fitness by 4', () => {
+        const pet = new Pet('Fido');
+
+        pet.fitness = 4;
+        pet.walk();
+
+        expect(pet.fitness).toEqual(8)
+    });
+
+    it('fitness reaches a maximum of 10', () => {
+        const pet = new Pet('Fido');
+
+        pet.fitness = 8;
+        pet.walk();
+
+        expect(pet.fitness).toEqual(10);
+    });
+});
+
+describe('feed', () => {
+    it('throws an error if pet is not alive', () => {
+        const pet = new Pet('Fido')
+
+        pet.age = 30;
+
+        expect(() => pet.feed()).toThrow('Your pet is no longer alive')
+    });
+
+    it('decreases hunger by 3', () => {
+        const pet = new Pet('Fido');
+
+        pet.hunger = 6;
+        pet.feed();
+
+        expect(pet.hunger).toEqual(3);
+    });
+
+    it('hunger reaches a minimum of 0', () => {
+        const pet = new Pet('Fido');
+
+        pet.hunger = 2;
+        pet.feed();
+
+        expect(pet.hunger).toEqual(0);
+    });
+});
+    describe('checkup', () => {
+        it('throws an error if pet is not alive', () => {
+            const pet = new Pet('Fido')
+    
+            pet.age = 30;
+    
+            expect(() => pet.checkup()).toThrow('Your pet is no longer alive')
+        });
+
+        it('returns string if fitness is 3 or less', () => {
+            const pet = new Pet('Fido');
+
+            pet.fitness = 2;
+            
+            expect(pet.checkup()).toEqual('I need a walk');
+        });
+
+        it('returns string if hunger is 5 or more', () => {
+            const pet = new Pet('Fido');
+
+            pet.hunger = 6;
+
+            expect(pet.checkup()).toEqual('I am hungry');
+        });
+
+        it('returns string if hungry and needs walk', () => {
+            const pet = new Pet('Fido');
+
+            pet.hunger = 9;
+            pet.fitness = 1;
+
+            expect(pet.checkup()).toEqual('I am hungry AND I need a walk')
+        });
+
+        it('returns string if dog is neither hungry or needs a walk', () => {
+            const pet = new Pet('Fido');
+
+            pet.hunger = 0;
+            pet.fitness = 10;
+
+            expect(pet.checkup()).toEqual('I feel great!')
+        });
+    });
+
+
+
+
+
```

Comment:
Should be a clear line between each describe block, and this whole block should be moved to the far left of the file since it isnt inside anything else. Also, theres no need for all the multiple clear lines at the end which can be removed:
```

describe('checkup', () => {
    it('throws an error if pet is not alive', () => {
        const pet = new Pet('Fido')

        pet.age = 30;

        expect(() => pet.checkup()).toThrow('Your pet is no longer alive')
    });

    it('returns string if fitness is 3 or less', () => {
        const pet = new Pet('Fido');

        pet.fitness = 2;
        
        expect(pet.checkup()).toEqual('I need a walk');
    });

    it('returns string if hunger is 5 or more', () => {
        const pet = new Pet('Fido');

        pet.hunger = 6;

        expect(pet.checkup()).toEqual('I am hungry');
    });

    it('returns string if hungry and needs walk', () => {
        const pet = new Pet('Fido');

        pet.hunger = 9;
        pet.fitness = 1;

        expect(pet.checkup()).toEqual('I am hungry AND I need a walk')
    });

    it('returns string if dog is neither hungry or needs a walk', () => {
        const pet = new Pet('Fido');

        pet.hunger = 0;
        pet.fitness = 10;

        expect(pet.checkup()).toEqual('I feel great!')
    });
});
```